data_managers/data_manager_iphop/data_manager/iphop_datamanager.xml

```diff
@@ -1,20 +1,26 @@
 <tool id="iphop_build_database" name="iPHoP" tool_type="manage_data" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@">
     <description>database builder</description>
     <macros>
-        <token name="@TOOL_VERSION@">1.3.3</token>
-        <token name="@VERSION_SUFFIX@">3</token>
+        <token name="@TOOL_VERSION@">1.4.1</token>
+        <token name="@VERSION_SUFFIX@">0</token>
         <token name="@PROFILE@">22.01</token>
     </macros>
     <requirements>
         <requirement type="package" version="@TOOL_VERSION@">iphop</requirement>
     </requirements>
     <command detect_errors="exit_code"><![CDATA[
-        mkdir -p '$out_file.extra_files_path'/db &&
+        mkdir tmp_download/ &&
         iphop download
-            --db_dir '$out_file.extra_files_path'/db
+            --db_dir tmp_download
             --db_version $db_version
             ## --full_verify  https://bitbucket.org/srouxjgi/iphop/issues/114/db-download-verification-error
             --no_prompt &&
+
+        ## download was done in a temp folder (containing tgz, md5 and extracted folder)
+        ## mv the folder contents to where Galaxy expects it, since the folder name depends on the download
+        ## it is determined with find
+        mkdir -p '$out_file.extra_files_path'/db &&
+        mv "\$(find tmp_download/ -mindepth 1 -maxdepth 1 -type d)"/* '$out_file.extra_files_path'/db/ &&
```

Comment:
So in the end, after Galaxy has moved the data the db folder will be named like the selected tool interface option value (not like the extracted inner folder). For example `iPHoP_db_Aug23_rw`, not `Aug_2023_pub_rw`.
I assume you've tested that this is not somehow an issue for the iphop tool then when trying to access the db?